bundles/net.tourbook/src/net/tourbook/messages.properties

```diff
@@ -3022,7 +3022,9 @@ Slideout_TMVOptions_Radio_ElapsedTime                    = &Elapsed Time
 Slideout_TMVOptions_Radio_MovingTime                     = &Moving Time
 Slideout_TMVOptions_Radio_RecordedTime                   = &Recorded Time
 
-Slideout_TourBlogOptions_Label_Title = Tour Blog Options
+Slideout_TourBlogOptions_Checkbox_ShowTourTags         = Show &Tour Tags
```

Comment:
* A checkbox text should not be display with titlecase -> `Show tour &tags`
* The mnemomic should be, when possible on the main word, here it is the `&tag`. The mnemonic in the above checkbox is not on `S&how` is is on `&hidden`

![image](https://github.com/mytourbook/mytourbook/assets/1283445/b6398f68-dc76-491a-bcf9-742708f8cfa5)

Reply:
Fixed.
However, I am not sure to understand the rule. If, like you say, it's supposed to be on the `main word`, why is it not on `markers` above ? Isn't marker the `main word` ?

Reply:
For me is the main word(s) the `default color` but I think that everybody can see this differently